libs/ui/src/lib/breadcrumbs/breadcrumbs.component.ts

```diff
@@ -18,6 +18,8 @@ export class BreadcrumbsComponent {
   /** Name of an icon in front of breadcrumb */
   @Input() iconName: CviIconName | null = null;
 
+  @Input() isGroupService = false;
```

Comment:
It is not clear what this prop does. In Storybook setting it to true doesn't have any effect. Would a dedicated story help?

Reply:
Adding to that, I think we should use more general naming here, since the "group" is only relevant to EBS project.